controllers/message-controller.ts

```diff
@@ -35,77 +37,21 @@ export default class MessageController implements MessageControllerI {
         if(MessageController.messageController === null) {
             MessageController.messageController = new MessageController();
 
-            app.post("/api/users/:uid/messages", MessageController.messageController.sendMessage);
-            app.get("/api/users/:uid/messages", MessageController.messageController.findAllMessagesSentByUser);
-            app.get("/api/users/:uid/received-messages", MessageController.messageController.findAllMessagesReceivedByUser);
-            app.delete("/api/messages/:mid", MessageController.messageController.deleteMessage);
-            app.put("/api/messages/:mid", MessageController.messageController.updateMessage);
-            app.delete("/api/users/:uid/messages", MessageController.messageController.deleteAllMessages);
+            app.post("/api/users/:uid1/messages/:uid2", MessageController.messageService.userSendsMessage);
+            app.get("/api/users/:uid/messages", MessageController.messageService.findAllMessagesSentByUser);
+            app.get("/api/users/:uid/receives", MessageController.messageService.findAllMessagesReceivedByUser);
+            app.delete("/api/messages/:mid", MessageController.messageService.userDeletesMessage);
+            app.put("/api/messages/:mid", MessageController.messageService.userUpdatesMessage);
+            app.delete("/api/users/:uid/messages", MessageController.messageService.deleteAllMessagesSentByUser);
+            app.delete("/api/users/:uid/receives", MessageController.messageService.deleteAllMessagesReceivedByUser);
```

Comment:
For now, let's not have the APIs to delete sent and received messages. I suspect this will cause some problem.